src/desktop/apps/experimental-app-shell/utils/userRequiredMiddleware.tsx

```diff
@@ -0,0 +1,22 @@
+import { getPageType } from "./getPageType"
+
+const USER_REQUIRED_PAGES = [
+  "orders",
+  "identity-verification",
+  "auction-registration",
+  "auction-registration2",
```

Comment:
#minor - mind alphabetizing this list?

Reply:
Done - to be clear I did not move the logic over from the bid confirmation screen as it has pages that would have been called `auction` & I felt like this was probably a more complex case than simply matching on the first url segment.

Reply:
I don't think it is, as that's exactly what we're doing now. If the logic is capable of greedily matching `*auction-*` and not the full string then we should use something that exactly matches. .